ui/apps/platform/src/Containers/ComplianceEnhanced/Coverage/components/StatusCountIcon.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Icon } from '@patternfly/react-core';
+import { BarsIcon, CheckCircleIcon, SecurityIcon } from '@patternfly/react-icons';
+import pluralize from 'pluralize';
+
+import IconText from 'Components/PatternFly/IconText/IconText';
+
+type StatusCountIconProps = {
+    text: string;
+    status: 'pass' | 'fail' | 'other';
+    count: number;
+};
+
+function getStatusIcon(status, count) {
+    let color = 'var(--pf-v5-global--disabled-color--100)';
+    if (count > 0) {
+        switch (status) {
+            case 'fail':
+                color = 'var(--pf-v5-global--danger-color--100)';
+                break;
+            case 'pass':
+                color = 'var(--pf-v5-global--primary-color--100)';
+                break;
+            case 'other':
+                color = 'var(--pf-v5-global--disabled-color--100)';
+                break;
+            default:
+                break;
```

Comment:
Especially if `'pass' | 'fail' | 'other'` string enumeration is a frontend type, then there is a recent idiom worthwhile to consider. We can ask David to give us a short tutorial tomorrow.
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Components/TableStateTemplates/TbodyUnified.tsx#L68-L69

I think it requires to define a string enumeration type and use it for the function argument, which seems worthwhile on its own merits.

Reply:
Defining the types in the parameters here makes sense but eslint will still complain about there not being a default case unfortunately.